server/utils/constant.ts

```diff
@@ -14,3 +14,8 @@ export const DEFAULT_NAVIGATION_ITEM = {
   slug: "main-navigation",
   visible: true,
 } as const;
+export enum RENDER_TYPES {
+  FLAT = 'FLAT',
+  TREE = 'TREE',
+  RFR = 'RFR'
+};
```

Comment:
[nitpick] meta of contemporary TS is to move away from TS specific constructs like `enum`, decorators, etc.

Reply:
@CodeVoyager  changed, if fine please resolve :)